inference-engine/src/low_precision_transformations/src/reshape.cpp

```diff
@@ -250,9 +250,18 @@ bool ReshapeTransformation::canBeTransformed(
             return false;
         }
     } else {
-        for (size_t i = 0; i < 2ul; ++i) {
-            if (inputShape[i] != outputShape[i]) {
-                return false;
+        auto isConstantShape = [&](const ngraph::Shape& shape) {
```

Comment:
@v-Golubev  @eshoguli  "isConstantShape" doesn't look like relevant name here. Just wondering shouldn't this check be named "isScalar" or "isPerTensor" and moved to common place since seems to be commonly used property of dequantization?

Reply:
@dmitry-gorokhov  I think this lambda can be replaced with (ngraph::shape_size == 1). shape_size returns the multiplication of the Shape components multiplied by 1.

Reply:
@v-Golubev Ok. I don't mind

Reply:
Done, thanks